src/api/routes/articleRouter.js

```diff
@@ -0,0 +1,17 @@
+import { Router } from 'express';
+import checkArticle from '../../middlewares/getOneArticle';
+import checkValidToken from '../../middlewares/checkValidToken';
+import bodyVerifier from '../../middlewares/validations/body.verifier';
+import ratingsController from '../controllers/rateArticleController';
+import validateInputs from '../../middlewares/validations/body.inputs';
+
+const articleRouter = new Router();
+
+articleRouter.post('/:slug/rating',
```

Comment:
check also if the body is empty. I created a middleware for that. @MCFrank16

Reply:
I did it man